### ISSUE TITLE
Website isn't deploying, needs rouge gem

### ISSUE THREAD

**Issue by MaxGhenis** [UBICenter/ubicenter.org#50]
Not sure how to fix this. 

See https://github.com/UBICenter/ubicenter.org/runs/1976243145?check_suite_focus=true

```
Successfully installed bundler-2.2.11
Parsing documentation for bundler-2.2.11
Installing ri documentation for bundler-2.2.11
Done installing documentation for bundler after 3 seconds
1 gem installed
Fetching gem metadata from https://rubygems.org/..........
Resolving dependencies...
Bundler found conflicting requirements for the Ruby version:
  In Gemfile:
    Ruby

    github-pages (~> 192) was resolved to 192, which depends on
      activesupport (= 4.2.10) was resolved to 4.2.10, which depends on
        Ruby (>= 1.9.3)

    github-pages (~> 192) was resolved to 192, which depends on
      Ruby (>= 2.3.0)

    jekyll (~> 3.7.4) was resolved to 3.7.4, which depends on
      Ruby (>= 2.1.0)

    github-pages (~> 192) was resolved to 192, which depends on
      listen (= 3.1.5) was resolved to 3.1.5, which depends on
        Ruby (>= 2.2.3, ~> 2.2)

    github-pages (~> 192) was resolved to 192, which depends on
      nokogiri (< 2.0, >= 1.8.2) was resolved to 1.11.1, which depends on
        Ruby (< 3.1.dev, >= 2.5)

    rake (~> 12.3.1) was resolved to 12.3.3, which depends on
      Ruby (>= 2.0.0)

Bundler could not find compatible versions for gem "rouge":
  In Gemfile:
    github-pages (~> 192) was resolved to 192, which depends on
      rouge (= 2.2.1)

    jekyll (~> 3.7.4) was resolved to 3.7.4, which depends on
      rouge (< 4, >= 1.7)
Error: Process completed with exit code 6.
```

**Comment by MaxGhenis** [comment]
We're using `actions/setup-ruby`:

https://github.com/UBICenter/ubicenter.org/blob/2cd3bcb16af3fdd606c4c96bdf931de48b972a95/.github/workflows/deploy.yml#L17

Per https://github.com/actions/setup-ruby we should move to https://github.com/ruby/setup-ruby. Might fix it.

**Comment by MaxGhenis** [comment]
I think this was fixed in #52
